test/resources/EphemeralKeys.spec.js

```diff
@@ -3,15 +3,15 @@
 const stripe = require('../../testUtils').getSpyableStripe();
```

Comment:
can we remove this global? it's confusing when helpers shadow the global with the parameter.